src/engine/Settings.cs

```diff
@@ -111,7 +111,14 @@ public enum StrainBarVisibility
     ///   enabled.
     /// </summary>
     [JsonProperty]
-    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0);
+    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0.6f);
+
+    /// <summary>
+    ///   The amount of blur applied to microbe backgrounds when distortion is on.
+    ///   enabled.
+    /// </summary>
```

Comment:
```suggestion
    ///   The amount of blur applied to microbe backgrounds when distortion is on.
```

I think that was a typo in the comment.